### ISSUE TITLE
Missing checksums for multiple packages and some packages missing from openSUSE server

### ISSUE THREAD

**Issue by fusion809** [fasheng/arch-deepin#108]
On a fresh Arch Linux VM I added the pacman repo /etc/pacman.conf, although I enabled both servers as I know from past experience missing packages (or max memory exceeding errors) are not uncommon with these servers. Then I ran `sudo pacman -Sy deepin-extra deepin` to get the error:
```sh
:: File /var/cache/pacman/pkg/dde-api-2.91.0-2-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (checksum)).

Do you want to delete it? [Y/n] n
:: File /var/cache/pacman/pkg/dde-control-center-2.90.3-3-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (checksum)).
Do you want to delete it? [Y/n] n
:: File /var/cache/pacman/pkg/deepin-nautilus-properties-3.14.2-2-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (checksum)).
Do you want to delete it? [Y/n] n
:: File /var/cache/pacman/pkg/libdeepin-webkit-1.8.2.2.20141124213858-8-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (checksum)).
Do you want to delete it? [Y/n] n
:: File /var/cache/pacman/pkg/lightdm-deepin-greeter-2.90.1-4-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (checksum)).
Do you want to delete it? [Y/n] n
error: failed to commit transaction (invalid or corrupted package)
Errors occurred, no packages were upgraded.
```

I then added a hashtag (#) before the second server (the UK-based one) and re-ran `sudo pacman -Sy deepin-extra deepin` and pressed `Y` to "Do you want to delete it" questions and re-ran `sudo pacman -Sy deepin-extra deepin` yet again and now I am getting the error:

```sh
error: failed retrieving file 'dde-api-2.91.0-2-x86_64.pkg.tar.xz' from download.opensuse.org : transfer closed with 1400757 bytes remaining to read
warning: failed to retrieve some files
 dde-control-center-...  1936.9 KiB   311K/s 00:06 [######################] 100%
 deepin-nautilus-pro...   438.8 KiB   362K/s 00:01 [######################] 100%
 libdeepin-webkit-1....     5.5 MiB   452K/s 00:12 [######################] 100%
 lightdm-deepin-gree...     2.9 MiB   428K/s 00:07 [######################] 100%
error: failed to commit transaction (unexpected error)
Errors occurred, no packages were upgraded.
```

**Comment by fasheng** [comment]
Well, this is a troublesome issue of OBS for Archlinux users that the the checksums will be out of date after package rebuild without version changed. Sometimes we will increase `pkgrel` manually to avoid such issues, but most of the cases, please install the packages manually through `pacman -U`.

**Comment by fusion809** [comment]
What all of them? Or just the corrupted ones? If it's just the corrupted ones well I just tried that and as soon as I run `sudo pacman -Sy deepin deepin-extra` to install the remaining packages it started to download the packages I just downloaded and installed manually and ran into the same checksum error. Downloading all the packages manually sounds like a mammoth task, surely there must be another way.
